src/FilesHelper.php

```diff
@@ -71,6 +71,9 @@ public static function getListOfLocalFilesByDir( string $dir ) : array {
         return $files;
     }
 
+    /**
+     * Ensure a given filepath has an allowed filename and extension.
+     */
     public static function filePathLooksCrawlable( string $file_name ) : bool {
```

Comment:
@Flynsarmy open to improving fn naming here, for sure!

Reply:
Updated through https://github.com/WP2Static/wp2static/pull/638/commits/8aca024add34736af406a565fcbf78f80a8420fa. Let me know if they're OK.

Reply:
minor edit I hate to ask for... sorry!